src/controller/CHIPDeviceController.cpp

```diff
@@ -1823,6 +1820,18 @@ void DeviceCommissioner::CloseBleConnection()
 }
 #endif
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+void DeviceCommissioner::ConnectWiFiPAFTransportToSelf()
+{
+    Transport::WiFiPAFBase & transport =
+        std::get<Transport::WiFiPAF<1>>(mSystemState->TransportMgr()->GetTransport().GetTransports());
+    if (!transport.IsWiFiPAFLayerTransportSetToSelf())
```

Comment:
Is this conditional needed? Why would the transport not include this internally?

Reply:
I will change it